tools/releaseBuild/Images/microsoft_powershell_windowsservercore/wix.psm1

```diff
@@ -0,0 +1,24 @@
+Import-Module "$PSScriptRoot\dockerInstall.psm1"
+
+function Install-WixZip
```

Comment:
Please add a comment why we switched to zip instead on MSI, due to it requiring older version of dotnet